release.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+get_latest() {
+  git fetch
+
+  git tag |
+    # drop `v` prefix
+    grep "^v" |
+    cut -c2- |
+
+    # sort by semantic version
+    sort -t "." -k1,1n -k2,2n -k3,3n |
+
+    # last
+    tail -n 1 |
+
+    # drop newline
+    tr -d '\n'
+}
+
+NEW="$1"
+
+if [ -z "$NEW" ]; then
+  echo "Usage  : bash release.sh <version>"
+  echo "Example: bash release.sh 1.2.3"
+  echo ""
+  echo "Fetching tags..."
+  latest="$(get_latest)"
+  echo -n "The current version is: $latest"
+
+  exit 1
+fi
+
+if [[ "$NEW" == v* ]]; then
+  echo "<version> must not start with \"v\""
+  exit 1
+fi
+
+echo "Checking for clean working tree..."
+if [[ "$(git diff --stat)" != "" ]]; then
+  echo "❌ Dirty working tree (try git stash)"
+  exit 1
+fi
+
+echo "Checking that we're on master..."
+if [[ "$(git symbolic-ref HEAD | tr -d '\n')" != "refs/heads/master" ]]; then
+  echo "❌ Not on master (try git checkout master)"
+  exit 1
+fi
+
+echo "Checking that master is up to date..."
+git fetch
+if [[ "$(git rev-parse master)" != "$(git rev-parse origin/master)" ]]; then
+  echo "❌ master is out of sync with origin/master (try git pull)"
+  exit 1
+fi
+
+git ls-tree -r HEAD --name-only -z examples | xargs -0 sed -i.sedbak "s/\"[0-9]*\.[0-9]*\.[0-9]*\" # LATEST/\"$NEW\" # LATEST/g"
+find . -name "*.sedbak" -print0 | xargs -0 rm
+
+git commit --all --message "release $NEW"
+git tag "v$NEW"
+git push --tags
+git push
+
+echo ""
+echo "✅ Released $NEW"
+echo ""
+echo "- Tags   : https://github.com/sourcegraph/terraform-google-executors/tags"
+echo "- Commits: https://github.com/sourcegraph/terraform-google-executors/commits/master"
+echo ""
+echo "Make sure CI goes green 🟢"
```

Comment:
Can we issue an `open` command to open the buildkite pipeline? (a la `src lsif upload -open`)

Reply:
Do you know how to get the build `#`?

Reply:
Probably good enough just to link to https://buildkite.com/sourcegraph/terraform-google-executors/builds?branch=master

Reply:
Lol I tried this and it works: https://buildkite.com/sourcegraph/terraform-aws-executors/builds/latest

Reply:
If you combine `/builds/latest` and `?branch=master` you get the _right_ thing :)

Reply:
Nice find 🔍  I'll add a Buildkite link